transmart-notifications/grails-app/services/org/transmart/notifications/QuerySetSubscriptionMailService.groovy

```diff
@@ -22,7 +23,8 @@ import org.transmartproject.core.userquery.SetType
 @Slf4j
 class QuerySetSubscriptionMailService {
 
-    def grailsApplication
+    @Value('${org.transmart.notifications.maxNumberOfSets}')
+    def maxNumberOfSets
```

Comment:
Why not just add type `Integer` here then?

Reply:
I did not specify it as an Integer, because when this property is not set, not found or it is set in some incorrect way, it fails with an ugly stacktrace in logs. With this assertion the error message looks better. I can also add a different way of error handling if you think it would be better.